packages/upload-api/test/access-client-agent.js

```diff
@@ -389,24 +397,27 @@ export const test = {
       agent,
       agent.issuer.did()
     )
-
+    // wait 100 milliseconds to let the email be sent
```

Comment:
`mail.take()` should take care of waiting until mail is there, so this seems obsolete

Reply:
ah so - I think this worked in the `access-api` testing infrastructure, but I had to hack around it to get this working in `upload-api` and didn't implement that particular bit - I think it would be straightforward to get it back in, so let me give that a shot...

Reply:
fixed!